app/status.py

```diff
@@ -60,11 +47,15 @@ def initialized(self) -> bool:
         ))
 
     def _get_scan_data(self) -> SimpleNamespace:
-        # Avoid circular import
-        from app.components.autoclose import autoclose_solved_posts  # noqa: PLC0415
-
-        next_scan = cast("dt.datetime", autoclose_solved_posts.next_iteration)
-
+        if not self.help_scan_loop:
+            return SimpleNamespace(
+                time_since="Disabled",
+                time_until_next="Disabled",
```

Comment:
1. Bold would make it more obvious to the user.
2. This isn't formatted to a location where one would normally expect a capital letter.

```suggestion
                time_since="**disabled**",
                time_until_next="**disabled**",
```